content/documentation/pages/6-recipes/4-kafka/1-ext-kafka-cluster-cf.md

```diff
@@ -83,27 +93,55 @@ With that flexibility, every stream application deployed through SCDF will autom
-    SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[foo]_CONNECTION_SPACE: # your cf connection properties
-    SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[foo]_CONNECTION_DOMAIN: # your cf connection properties
-    SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[foo]_CONNECTION_USERNAME: # your cf connection properties
-    SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[foo]_CONNECTION_PASSWORD: # your cf connection properties
-    SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[foo]_DEPLOYMENT_SERVICES:  # your cf connection properties
-    SPRING_PROFILES_ACTIVE: cloud
-    JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '{enabled: false}'
-    SPRING_APPLICATION_JSON: '{"spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.brokers":"foo0.broker.foo,foo1.broker.foo,foo2.broker.foo","spring.cloud.dataflow.applicationProperties.stream.spring.spring.cloud.stream.kafka.binder.jaas.options.username":"test","spring.cloud.dataflow.applicationProperties.stream.spring.spring.cloud.stream.kafka.binder.jaas.options.password":"bestest","spring.cloud.dataflow.applicationProperties.stream.spring.spring.cloud.stream.kafka.binder.jaas.loginModule":"org.apache.kafka.common.security.plain.PlainLoginModule",
-    "spring.cloud.skipper.client.serverUri": "https://<SKIPPER_URI>/api"}'
+  SPRING_PROFILES_ACTIVE: cloud
+  JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '{enabled: false}'
+  SPRING_APPLICATION_JSON: |-
+    {
+        "spring.cloud": {
+            "dataflow.task.platform.cloudfoundry": {
+                "accounts": {
+                    "foo": {
+                        "connection": {
+                            "url": <api-url>,
+                            "org": <org>,
+                            "space": <space>,
+                            "domain": <app-domain>, 
+                            "username": <email>, 
+                            "password": <password>,
+                            "skipSslValidation": true
+                        },
+                        "deployment": {
+                            "services": <comma delimited list of service>"
+                        }
+                    }
+                }
+            }, 
+            "stream": { 
+                "kafka.binder": {
+                    brokers": "foo0.broker.foo,foo1.broker.foo,foo2.broker.foo",
+                    "jaas": {
+                        "options": {
+                            "username": "test",
+                            "password":"bestest"
+                        },
+                        "loginModule":"org.apache.kafka.common.security.plain.PlainLoginModule"
+                    }
+                },
+                "bindings.output.destination":"fooTopic"
+            },
+            "skipper.client.serverUri": "https://<SKIPPER_URI>/api"
```

Comment:
We could follow the same format that we do in the previous sections for consistency.

> SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI: http://your-skipper-server-uri/api

Reply:
Do you mean the value or use separate env var for this?

Reply:
Cool with either of the choices, but I was only suggesting that we could follow 1 pattern consistently everywhere.